storage_server/report_server.py

```diff
@@ -526,11 +526,16 @@ def addSuppressBug(self, run_id, bugs_to_suppress):
         try:
             suppressList = []
             for bug_to_suppress in bugs_to_suppress:
-                suppress_bug = SuppressBug(run_id,
-                                           bug_to_suppress.bug_hash,
-                                           bug_to_suppress.file_name,
-                                           bug_to_suppress.comment)
-                suppressList.append(suppress_bug)
+                res = self.session.query(SuppressBug) \
+                    .filter(SuppressBug.hash == bug_to_suppress.bug_hash) \
```

Comment:
Yes, copy paste in the same file is still a problem, we could not come up with a good enough hash to deal with that.